Call for mentors (#414)
ropensci/roweb3: content/blog/2022-10-20-call-for-mentors-champions-program/index.md

* Create a new package;

* Go through the review process with an R package they have already developed;

* Serve as a reviewer for the rOpenSci software peer review system.

Mentors may also support Champions with delivering other activities including: giving a talk, writing blog posts, and teaching other community members.

### Role description

Mentors will play a vital role in the rOpenSci Champions Program by helping select the [first cohort of Champions](/blog/2022/09/22/launch-champions-program/#how-to-apply) and advising and inspiring their mentees. Mentors will also __connect__ their mentees to people, programs, and organizations; __recommend__, resources, readings, training, and experiences; and provide __feedback__ on the mentee's project.

We are seeking mentors from the rOpenSci community who have:

* Demonstrated an understanding and ability to work openly and collaboratively;

* The willingness to guide and teach others;

* Previous experience with any of the following: 

>   * Developing, maintaining or contributing to any rOpenSci package or package which have gone through rOpenSci review, or

Reviewer: ```suggestion
  * Developing, maintaining or contributing to any rOpenSci package or package which has gone through rOpenSci review, or
```

---

> title: rOpenSci seeks Mentors 

Reviewer 2: ```suggestion
title: Become a Mentor for rOpenSci Champions!
```

or something else more "active"?

Author: .Yes!

author: Yanina Bellini Saibene
date: '2022-10-20T00:00:01-07:00'
slug: mentors-champions-program
tags:
  - community
  - help
  - beginner
  - welcome
  - diversity
  - champions program
feature: true  

> description: "rOpenSci seeks mentors to support our inaugural cohort of rOpenSci Champions. Could you offer insight and advice to our selected champions? Learn the details and express your interest." 

Reviewer 2: ```suggestion
description: "rOpenSci is seeking mentors to support our inaugural cohort of rOpenSci Champions. Could you offer insight and advice to our selected champions? Learn the details and express your interest." 
```

> description: "rOpenSci is seeking mentors to support our inaugural cohort of rOpenSci Champions. Could you offer insight and advice to our selected champions? Learn the details and express your interest." 

Reviewer: ```suggestion
description: "rOpenSci is seeking mentors to support our inaugural cohort of rOpenSci Champions. Could you offer insight and advice to our selected Champions? Learn the details and express your interest!" 
```

---

> ## rOpenSci seeks mentors to support our inaugural cohort of rOpenSci Champions.

Reviewer 2: ```suggestion
## rOpenSci is seeking mentors to support our inaugural cohort of rOpenSci Champions.
```

> ## rOpenSci is seeking mentors to support our inaugural cohort of rOpenSci Champions.

Author: ```suggestion
## rOpenSci is seeking mentors to support our inaugural cohort of rOpenSci Champions!
```

The [rOpenSci Champions Program](/champions/) is for _people from historically and systematically excluded groups_ who are interested in contributing to rOpenSci and the broader ecosystem of open source and open science communities. After receiving training, mentors will work with Champions to develop their project, which can be one of the following options:

> * Create a new package

Reviewer 2: ```suggestion
* Create a new package;
```

> * Go through the review process with an R package they already have developed

Reviewer 2: ```suggestion
* Go through the review process with an R package they already have developed;
```

> * Serve as a reviewer for the rOpenSci  software peer review system

Reviewer 2: ```suggestion
* Serve as a reviewer for the rOpenSci software peer review system.
```

> Mentors may also support Champions with delivering other activities including; giving a talk, writing blog posts, and teaching other community members.

Reviewer 2: ```suggestion
Mentors may also support Champions with delivering other activities including: giving a talk, writing blog posts, and teaching other community members.
```

> Mentors may also support Champions with delivering other activities including: giving a talk, writing blog posts, and teaching other community members.

Author: We expect they will help with the main project but not with all the outreach activities (this is probably more my job). But mentors can advise on events to participate in or topics to write/ prepare a talk, for example.  That is why the _may_ word.

Is that make sense?

> Mentors may also support Champions with delivering other activities including: giving a talk, writing blog posts, and teaching other community members.

Reviewer: Yup, definitely! "may" makes perfect sense in this context

### Role Description

> Mentors will play a vital role in the rOpenSci champions program by selecting the first cohort of Champions and advising and inspiring their mentees. Mentors will also __connect__ their mentees to people, programs, organizations; __recommend__, resources, readings, training, experiences and provide __feedback__ about the mentee's project.

Reviewer 2: ```suggestion
Mentors will play a vital role in the rOpenSci champions program by helping select the [first cohort of Champions](/blog/2022/09/22/launch-champions-program/#how-to-apply) and advising and inspiring their mentees. Mentors will also __connect__ their mentees to people, programs, organizations; __recommend__, resources, readings, training, experiences and provide __feedback__ about the mentee's project.
```

> We are seeking Mentors who are rOpenSci community members who have:

Reviewer 2: how is "rOpenSci community members" defined?

Reviewer 2: oh I see, in one of the items below, sorry.

> * Demonstrated an understanding and ability to work openly and collaboratively

Reviewer 2: ```suggestion
* Demonstrated an understanding and ability to work openly and collaboratively;
```

> * The willingness to guide and teach others

Reviewer 2: ```suggestion
* The willingness to guide and teach others;
```

* Previous experience with any of the following: 

  * Developing, maintaining or contributing to any of our packages or the packages we review, or
  
  * Participating in the rOpenSci review process, or 
  

>   * Package development and/or peer code review

Reviewer 2: ```suggestion
  * Package development and/or peer code review.
```

>   * Package development and/or peer code review.

Author: I think we are open to people from other communities, especially developing a package option.  I think is hard to ask someone that doesn't know our review process to mentor on that topic.  So, how can we make this clear?

A rOpenSci member is anyone that contributes to our community in any way.

>   * Package development and/or peer code review.

Reviewer: Perhaps simply add a question to the AirTable form: 
- "How do you participate in the rOpenSci community?" (please answer in 1 or 2 sentences), or,
- "Do you identify as part of the rOpenSci community? How do you participate?"
- 
Then people are free to self-identify and we can omit mentors who say "I don't" :grin: as opposed to answers like "I read the rOpenSci blog and follow rOpenSci on Twitter", or "I use lots of rOpenSci packages in my work", etc.

That way we just leave this list as is and vet whether or not someone is a community member by how they respond when signing up?

> Mentors will gain tutoring experience and skills, be acknowledged on the rOpenSci website and social media channels, and make new connections across the champions networks. This is a great way to give back to the rOpenSci community by mentoring our selected champions and helping to shape this program by providing feedback on this pilot year. 

Author: It is a good question. Let's see how the text reads with the changes and we can remove it if it is still redundant.

### Expectations

The total time commitment is expected to be approximately 1-2 hours per month from November 2022 - December 2023 to complete the following tentative program timeline.

|Date|Event|Detail|Duration|
|----|-----|------|--------|

> |November, 2022|Select Champions |Review, rank and select 1-2 reviewed projects to mentor|2 h|

Reviewer 2: ```suggestion
|November, 2022|Select Champions |Review, rank and select 1-2 reviewed projects to mentor.|2 h|
```

|December 2022|Kick-off event|Learn about the program, review the program expectations, and address questions. Meet mentees.|1.5 h|
|February 2023|Training|How to mentor and logistics on the program.|1 h|

> |March 2023|Cohort meeting: Presentation of projects|Champions' 5 minutes presentations of their project (package review or development) and get feedback.|1.5 h|

Reviewer 2: ```suggestion
|March 2023|Cohort meeting: Presentation of projects|Champions give 5 minutes presentations of their project (package review or development) and get feedback.|1.5 h|
```

|March-July 2023|Meeting with mentee|Provide feedback and guidance about mentee project.  Connect with other experts, mentors or community member if necessary. Recommend training, resources for the mentee project.|1 meeting per month 0.5 h|
|Jun 2023|Cohort meeting: Reflections on projects|Champions will be invited to share takeaways from their projects with the cohort.|1.5 h|
|August - November 2023|Meeting with mentee|Provide feedback and guidance about mentee project.  Connect with other experts, mentors or community member if necessary. Recommend training, resources for the mentee project. |1 meeting per month 0.5 h|
|September 2023|Cohort meeting: Reflections on events|Champions will be invited to share takeaways from their events with the cohort.|1.5 h|
|December 2023|Cohort meeting: Reflections on program|Mentors will be invited to share takeaways from the program and help to define future roles for Champions.|1.5 h

### Benefits of being a mentor

By becoming an rOpenSci mentor you will:

> * Empower members of our community by sharing your expertise with others; 

Reviewer 2: should this item mention "historically and systematically excluded groups"?

Author: Yes

> * Empower members of our community by sharing your expertise with others; 

Author: ```suggestion
* Empower members of historically and systematically excluded groups by sharing your expertise with others; 
```

* Connect with other mentors and with the champion community, expanding your professional network;

> * Be the bridge between your mentee and other community members;

Author: Oh, that is my bias.  I love to be the bridge between people, and I think learning to do that is a skill. 
I like what you add instead.

> * Be the bridge between your mentee and other community members;

Reviewer: You could leave it in and just rephrase it as you said, a skill you'll develop! Something like:  "Learn how to be a bridge between a mentee and other community members"

> * Gain exposure, recognition and acknowledgment for your work as a mentor in rOpenSci spaces (website, socal media, events)

Reviewer: ```suggestion
* Gain exposure, recognition and acknowledgment for your work as a mentor in rOpenSci spaces (website, social media, events)
```

* Help rOpenSci and the wider R community grow;
* Co-develop this program and other rOpenSci projects by sharing feedback;

Mentors will participate in a training about mentoring and program logistics as part of their onboarding process and have the support of the Community Manager throughout the program.

### How to apply

> Please, fill out the [application form](https://airtable.com/shrlrLdQbb4wphosK) ([this template shows the information we will require](/champions/files/mentors_champions_template)) before __Monday Nov 7, 2022__. The application forms have to be completed _in English_.

Reviewer 2: ```suggestion
Please, fill out the [application form](https://airtable.com/shrlrLdQbb4wphosK) ([this template shows the information we will require](/champions/files/mentors_champions_template)) before __Monday November 7, 2022__. The application forms have to be completed _in English_.
```

## Do you have any questions? 

Please, contact our Community Manager by [email](mailto:yabellini@ropensci.org) or [book a 15 minutes meeting with her](https://calendly.com/yabellini-ropensci/15min).

> We will have a [Community Call October, 24](/commcalls/oct2022-champions/) to share tips for submitting your Champions Program application and a [Coworking space on November, 1](/events/coworking-2022-11/) to help you with your application to the Champions Program.

Reviewer 2: ```suggestion
We will have a [Community Call October, 24](/commcalls/oct2022-champions/) to share tips for submitting your Champions Program application and a [Coworking slot on November, 1](/events/coworking-2022-11/) to help you with your application to the Champions Program.
```

> We will have a [Community Call on October 24th](/commcalls/oct2022-champions/) to share tips for submitting your Champions Program application and a [Coworking slot on November 1st](/events/coworking-2022-11/) to help you with your application to the Champions Program.

Author: ```suggestion
We will have a [Community Call on October 24th](/commcalls/oct2022-champions/)to explain and explore the benefits of Champions Programs and a [Coworking slot on November 1st](/events/coworking-2022-11/) to answer any questions you might have about mentoring in this program.
```

Author: as suggested by Steffi.
